lib/fog/aws/version.rb

```diff
@@ -1,5 +1,5 @@
 module Fog
   module AWS
-    VERSION = "1.4.1"
+    VERSION = "1.4.2"
```

Comment:
Also, if you could leave this out. We'll make this change when we cut a release.